src/common/components/SubContentBox.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+
+class SubContentBox extends Component {
+  render() {
+    const { children, title } = this.props;
```

Comment:
is the ``title`` required? what if we  pass a ``children`` without ``title``

Reply:
No it's not, you can check the `propTypes` and test for the case

Reply:
what I mean is will render and empty ``<h3>`` do we have such a use case? otherwise I'd put ``title`` as required

Reply:
empty `h3` is not breaking the UI.